tests/dataset/conftest.py

```diff
@@ -0,0 +1,20 @@
+import numpy as np
+import pytest
+
```

Comment:
we will want to use this fixture in other test files like test_models.py for example, so probably want to put it in a higher level conftest